src/App.js

```diff
@@ -1,13 +1,23 @@
 import React from "react";
-
+import Heading from "./components/Heading";
 import Bookings from "./Bookings";
+import TouristInfoCards from "./components/TouristInfoCards";
+import Footer from "./components/Footer";
 import "./App.css";
-
+import Restaurant from "./Restaurant";
+const info = [
+  "Address: 123 Fake Street, London, E1 4UD",
+  "Email: hello@fakehotel.com",
+  "Telephone Number: 0123 456789"
+];
 const App = () => {
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <Heading />
+      <TouristInfoCards />
       <Bookings />
+      <Restaurant />
+      <Footer arrayInfo={info} />
```

Comment:
Do you think `arrayInfo` is the most descriptive name? Does it help us understand what is in that prop?

Reply:
I have updated it with a descriptive name such as hotelContactInfo , thanks Ali.